obs-browser-plugin.cpp

```diff
@@ -221,12 +235,12 @@ static obs_properties_t *browser_source_get_properties(void *data)
 	return props;
 }
 
-static CefRefPtr<BrowserApp> app;
 
-static void BrowserInit(void)
+static void BrowserInit(obs_data_t *settings_obs, obs_source_t *source)
```

Comment:
Why is the source being passed? It seems like we only need the settings no?

Reply:
done